.gitignore

```diff
@@ -3,3 +3,4 @@ repos/
 sonar-agent.key
 .id_rsa
 .vault-token
+.idea
```

Comment:
you should be able to add that to your personal global gitignore

Reply:
actually I hadn't linked my global one which is what I realized after I added this lol

Reply:
haha, I figured you missed that on your migration 😀